audacity.sls

```diff
@@ -1,33 +1,18 @@
+{% set versions = ['2.2.1', '2.1.0', '2.0.6'] %}
```

Comment:
2.2.1 isn't available on sourceforge. It's on fosshub now? The site sends you to:
https://www.fosshub.com/Audacity.html/audacity-win-2.2.1.exe

Reply:
Might need two loops... One for 2.1.2 and greater and another for the older versions